README.md

```diff
@@ -41,7 +41,7 @@ This repository is a merger of the formerly separate GoogleTest and GoogleMock
 projects. These were so closely related that it makes sense to maintain and
 release them together.
 
-### Getting Started
+### | Getting Started |
```

Comment:
Why are there | chars around Getting Started?

Reply:
it indicate that "Getting Started" is a heading in a larger body of text, also drawing attention to it.

Reply:
Please take a look at the preview of the file: https://github.com/google/googletest/blob/a7fa9b37082993107a98181e631cafcdbe484f22/README.md#-getting-started-

I don't think the markdown is rendering as you intend.